changelog/changelog_3.0.0-4.0.0.txt

```diff
@@ -1,21 +1,56 @@
+20.08.2024
+Description:
+    - Reference Domain Info was added
+Required integration changes:
+    - None, however, users are encouraged to use Reference Domain Info
```

Comment:
This should reflect to some extent what the domain info is. For integration changes, I'd write where this is needed (signals, device domain), and maybe some pointers for references.